test/data/Coverage.jl

```diff
@@ -1,81 +1,5 @@
 #######################################################################
-# Coverage.jl
-# Take Julia test coverage results and bundle them up in JSONs
-# https://github.com/IainNZ/Coverage.jl
+# This file is not executed, but is used as comparison input
```

Comment:
Huh, most of `test/data/Coverage.jl` is removed in the first commit -- why? Also, `test/data/Coverage.jl.cov` is not updated to match. This seems wrong to me, but perhaps I am missing something?

Reply:
They don't need to match. I just removed most of the content because it's distracting (confuses search/grep/me to have a large quantity of almost-code)

Reply:
So I guess you point here is that `r = process_file(joinpath(datadir,"Coverage.jl"))` only really cares about `Coverage.jl.cov` and not about `Coverage.jl` itself (other than that `Coverage.jl` needs to exist?

But then the comment should say that. The current says that it is used as "comparison input", but if that was true, then surely removing most of the content of the file would break that comparison. So, why keep any content, other than a comment? E.g.
```
# This file is not used; it only exists as a placeholder so that `process_file` can
# be called on it in order to read `Coverage.jl.cov`.
```
or something like that.